stix_shifter/src/modules/dummy/dummy_data_mapping.py

```diff
@@ -0,0 +1,31 @@
+from os import path
+import json
+from stix_shifter.src.exceptions import DataMappingException
+
+
+def _fetch_mapping():
+    try:
+        basepath = path.dirname(__file__)
+        filepath = path.abspath(
+            path.join(basepath, "json", "from_stix_map.json"))
+
+        map_file = open(filepath).read()
+        map_data = json.loads(map_file)
+        return map_data
+    except Exception as ex:
+        print('exception in main():', ex)
+        return {}
+
+
+class DummyDataMapper:
+
+    def __init__(self, options):
+        self.mapping_json = options['mapping'] if 'mapping' in options else {}
+
+    def map_field(self, stix_object_name, stix_property_name):
+        self.map_data = self.mapping_json or _fetch_mapping()
```

Comment:
why not call `_fetch_mapping()` inside the constructor instead?
if `self.mapping_json` is not set then you call  `_fetch_mapping()` every call of `map_field `

besides, `self.mapping_json` will never ever be not set now as you set it to empty object `{}`